tests/test_filters.py

```diff
@@ -95,3 +96,59 @@ def test_multiple_urls():
 def test_no_links_no_change():
     text = 'There are no Greek Γ Δ Ε Ζ Η Θ Ι Κ Λ links.'
     assert format_links(text) == text
+
+
+def test_timesince():
+    now = datetime.utcnow()
+    times = [
+        now,
+
+        now - timedelta(seconds=1),
+        now - timedelta(seconds=2),
+
+        now - timedelta(minutes=1),
+        now - timedelta(minutes=2),
+
+        now - timedelta(hours=1),
+        now - timedelta(hours=2),
+
+        now - timedelta(days=1),
+        now - timedelta(days=2),
+
+        now - timedelta(days=7),
+        now - timedelta(days=14),
+
+        now - timedelta(days=30),
+        now - timedelta(days=60),
+
+        now - timedelta(days=365),
+        now - timedelta(days=365*2),
+    ]
+
+    texts = [
+        'just now',
```

Comment:
@sarneaud  Added 'now' parameter in timesince which might come handy in other cases as well. Thanks for the suggestion :+1: